tool/tsh/tsh.go

```diff
@@ -181,6 +181,8 @@ type CLIConf struct {
 	DaemonCertsDir string
 	// DatabaseService specifies the database proxy server to log into.
 	DatabaseService string
+	// DaemonPrehogAddr is the URL where prehog events should be submitted.
+	DaemonPrehogAddr string
```

Comment:
Let's put it below `DaemonCertsDir` and above `DatabaseService`.

Reply:
Right, I meant to put it like that.